scripts/lint.ts

```
@@ -0,0 +1,559 @@
+
+	try {
+		const graph = getDependencyGraph(sourceRoot, entryPoints, runner);
+		const targetRelative = relative(sourceRoot, absPath).replaceAll("\\", "/");
+		return invertGraph(graph, targetRelative).map((file) => join(sourceRoot, file));
+	} catch {
+		return [];
+	}
+}
+
+/* v8 ignore start -- CLI entrypoint */
+const IS_CLI_INVOCATION = process.argv[1]?.endsWith("scripts/lint.ts") === true;
```

ESLint

[error] Unsafe call of a type that could not be resolved.

(ts/no-unsafe-call)